### ISSUE TITLE
error rendering unknown string as html

### ISSUE THREAD

**Issue by alison985** [mozilla/redash#222]
https://bugzilla.mozilla.org/show_bug.cgi?id=1393838#c6

related to last release

**Comment by alison985** [comment]
PR #224

**Comment by madalincm** [comment]
I'm not sure how to test this in stage as there is no Presto data source in there.  I tried to find other data sources/tables where I could test this issue but I was unsuccessful so far.  I am open to suggestions on how we can test this in stage. If nothing comes up I will postpone verifying this bug after the push and verify it in stage using the Presto data source.

**Comment by alison985** [comment]
@madalincm This went to prod with M9.1.

**Comment by madalincm** [comment]
I have used the query from here https://sql.telemetry.mozilla.org/queries/24136/source in production and it appears I can still reproduce the issue. This is how the downloaded csv is looking:
![htmlissue](https://user-images.githubusercontent.com/15685441/30032530-1bc8d784-919f-11e7-89e1-c7818fd5c8c2.png)

I have created a new query using the same code as the above one, but the issue is still reproduceing.
Used test query: https://sql.telemetry.mozilla.org/queries/30107/source

Reopening the issue.
